naomi_osullivan/week_01/friday/mta6line.js

```diff
@@ -0,0 +1,88 @@
+var six = ['Astor Place', 'Union Sq6', '23rd6', '28th6', '33rd', 'Grand Central'];
+
+var theSix = function (start, finish, line) {
+  var stopsArray = [];
+//for loop
+if (start < finish) { //w/out positive loop works with neg loop works
```

Comment:
I think this is causing your trouble. You need to be comparing the _indexes_ of the start and finish stations, not the `start` and `finish` variables themselves (which just contain strings, meaning that when you test whether `start < finish` you are just asking which string comes first alphabetically). 

Try this again with `if( line.indexOf(start) < line.indexOf(finish) )` and see how you go.